modules/ombi.py

```diff
@@ -0,0 +1,540 @@
+                return False
+        except Exception, e:
+            logger.error('Exception thrown via %s Response %s' % (u, str(e)))
+            return False
+
+    @cherrypy.expose()
+    @require(member_of(htpc.role_admin))
+    def test(self, **kwargs):
+        # Fuller version of ping(). Includes auth test.
+        if self.ping(**kwargs):
+            logger.debug('ping ok, now trying credentials')
+            url = 'api/v1/Settings/about'
+            res = self._ombi_get(url)
+            if res != 'False':
+                logger.debug('Successful Ombi test')
+                return 'True'
+            logger.error('Ombi test failed: %s' % res)
+        return
+
+    @cherrypy.expose()
+    @require(member_of(htpc.role_admin))
+    def content_sync(self, source, mode, **kwargs):
+        if source == 'plex':
+            if self.get_plex_enabled() == 'True':
+                if mode == 'full':
+                    url = 'api/v1/Job/plexcontentcacher'
+                else:
+                    url = 'api/v1/Job/plexrecentlyadded'
+        if source == 'emby':
+            if self.get_emby_enabled() == 'True':
+                url = 'api/v1/Job/embycontentcacher'
+        d = self._ombi_post(url)
+        if d != 'False':
+            logger.debug('%s %s sync triggered' % (source, mode))
+            return 'True'
+        logger.debug('%s %s sync failed' % (source, mode))
+        return 'False'
+
+    @cherrypy.expose()
+    @require()
+    def get_emby_enabled(self):
+        d = self._ombi_get('api/v1/Settings/emby')
+        if d != 'False':
+            return str(d['enable'])
+        logger.debug('Couldnt get Ombi settings for Emby')
+        return 'False'
+
+    @cherrypy.expose()
+    @require()
+    def get_plex_enabled(self):
+        d = self._ombi_get('api/v1/Settings/plex')
+        if d != 'False':
+            return str(d['enable'])
+        logger.debug('Couldnt get Ombi settings for Plex')
+        return 'False'
+
+    def _ombi_get(self, url):
+        """
+        Combined function to make all swagger API GET calls.
+        Builds the url, authenticates and grabs token if there isn't one.
+        :url: api endpoint
+        :return: the json response content or 'False'
+        """
+        if self.auth() == 'False':
+            logger.error('GET request died - auth failed')
+            return 'False'
+        ssl = 's' if htpc.settings.get('ombi_ssl', False) else ''
+        ip = htpc.settings.get('ombi_host')
+        port = htpc.settings.get('ombi_port')
+        u = 'http%s://%s:%s/%s' % (ssl, ip, port, url)
+        h = dict()
+        h.update({ 'Authorization': self._token})
+        r = requests.get( u, headers=h )
+        if r.status_code == 200:
+            d = r.json()
+            return d
+        logger.error('Request failed %s %s %s' % (u, str(r.status_code), r.reason))
+        return 'False'
```

Comment:
Whats up with the bools like strings? why not use return False instead of 'False'

Reply:
Can I pass a 'false' bool back via json, between the .py back end and the client/browser .js at the front end? My testing indicated I couldn't, that I needed to be either passing back a string or a legit json response. 
I'll take a look at some of the other modules, see if I can come up with a better way.